hibernate-core/src/main/java/org/hibernate/engine/spi/SharedSessionContractImplementor.java

```diff
@@ -167,7 +167,10 @@ default void checkOpen() {
 	 * @apiNote This "timestamp" need not be related to timestamp in the Java Date/millisecond
 	 * sense.  It just needs to be an incrementing value.  See
 	 * {@link CacheTransactionSynchronization#getCurrentTransactionStartTimestamp()}
+	 *
+	 * @deprecated no longer supported, when 2LC is enabled {{@link CacheTransactionSynchronization#getCachingTimestamp()}} can be used.
```

Comment:
It took me a while to figure out that "2LC" meant second-level cache. Could we use spell it out here please?